src/pages/docs/chat/rooms/message-reactions.mdx

```diff
@@ -43,6 +43,50 @@ await room.messages.reactions.send(message, {
     count: 100,
 });
 ```
+
+```swift
+// Add a 👍 reaction using the default type
+await room.messages.reactions.send(to: message.serial, params: .init(reaction: "👍"))
```

Comment:
@umair-ably I think we should change this to `name`, to have consistency across SDKs